Feat: Add Hide/show toggle feature for all password fields

kahanikaar opened the pull request:
# Description
Password toggle feature added to view the entered password to verify whether correct or not.

Fixes #1077 

# Type of Change:
- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)


# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

Kajol-Kumari reviewed (changes requested):
@kahanikaar please attach the mock/GIF of the feature in the PR description.

kahanikaar commented:
> @kahanikaar please attach the mock/GIF of the feature in the PR description.

Screenshots of the feature added. Will this work? @Kajol-Kumari

Kajol-Kumari commented on vms/vms/static/vms/js/passwordtoggle.js:
```diff
@@ -0,0 +1,7 @@
+  
+  var input = $("#id_password");
+  input.attr('type') === 'password' ? input.attr('type','text') : input.attr('type','password')
+});
```
please leave a blank line here.

kahanikaar replied:
Changes are made as per asked. Do check @Kajol-Kumari

satya7289 commented:
@kahanikaar please update your description of the PR, Put the gif/ screenshot or video showing how this feature is working, in the `How has this been tested` section.
Also, please follow the commit message guidelines otherwise, it would not get merge.(also squash multiple commits into the single commit)

Kajol-Kumari reviewed (dismissed):
LGTM :+1: 
Please squash the commits so that It can be taken forward for merging.

kahanikaar commented:
> LGTM +1
> Please squash the commits so that It can be taken forward for merging.

Squashed the commits and changed the commit message. Do review. @Kajol-Kumari @satya7289
